2020/08/11/17:55
업데이트를 시작하겠습니다.
워프레임 업데이트 중입니다.....
Traceback (most recent call last):
  File "/workspace/crawling/python/warframes_update.py", line 112, in <module>
    save_data = warframe_crawling(item, path, path_0)
  File "/workspace/crawling/python/warframes_update.py", line 94, in warframe_crawling
    make_file(get_item, get_path)
  File "/workspace/crawling/python/warframes_update.py", line 84, in make_file
    value = pandas_value.pandas_value(get_item, 'warframe')
  File "/workspace/crawling/python/pandas_value.py", line 22, in pandas_value
    result_mnd = find_mnd.find_mmd(path)
NameError: name 'find_mnd' is not defined
무기 업데이트 중입니다.....
Traceback (most recent call last):
  File "/workspace/crawling/python/weapons_update.py", line 98, in <module>
    save_data = warframe_crawling(item, path, path_0)
  File "/workspace/crawling/python/weapons_update.py", line 79, in warframe_crawling
    make_file(get_item, get_path)
  File "/workspace/crawling/python/weapons_update.py", line 69, in make_file
    value = pandas_value.pandas_value(get_item, 'weapon')
  File "/workspace/crawling/python/pandas_value.py", line 22, in pandas_value
    result_mnd = find_mnd.find_mmd(path)
NameError: name 'find_mnd' is not defined
기타 무기 업데이트 중입니다.....
Traceback (most recent call last):
  File "/workspace/crawling/python/weapons_etc_update.py", line 98, in <module>
    save_data = warframe_crawling(item, path, path_0)
  File "/workspace/crawling/python/weapons_etc_update.py", line 79, in warframe_crawling
    make_file(get_item, get_path)
  File "/workspace/crawling/python/weapons_etc_update.py", line 69, in make_file
    value = pandas_value.pandas_value(get_item, 'weapon')
  File "/workspace/crawling/python/pandas_value.py", line 22, in pandas_value
    result_mnd = find_mnd.find_mmd(path)
NameError: name 'find_mnd' is not defined
모드 업데이트 중입니다.....
Traceback (most recent call last):
  File "/workspace/crawling/python/mods_update.py", line 101, in <module>
    save_data = warframe_crawling(item, path, path_0)
  File "/workspace/crawling/python/mods_update.py", line 82, in warframe_crawling
    make_file(get_item, get_path)
  File "/workspace/crawling/python/mods_update.py", line 72, in make_file
    value = pandas_value.pandas_value(get_item, 'mod')
  File "/workspace/crawling/python/pandas_value.py", line 22, in pandas_value
    result_mnd = find_mnd.find_mmd(path)
NameError: name 'find_mnd' is not defined
아케인 업데이트 중입니다.....
Traceback (most recent call last):
  File "/workspace/crawling/python/etcs_update.py", line 101, in <module>
    save_data = warframe_crawling(item, path, path_0)
  File "/workspace/crawling/python/etcs_update.py", line 82, in warframe_crawling
    make_file(get_item, get_path)
  File "/workspace/crawling/python/etcs_update.py", line 72, in make_file
    value = pandas_value.pandas_value(get_item, 'etc')
  File "/workspace/crawling/python/pandas_value.py", line 22, in pandas_value
    result_mnd = find_mnd.find_mmd(path)
NameError: name 'find_mnd' is not defined
결산 중입니다.....
2020/08/11/17:56
업데이트를 마쳤습니다.
